routes/article.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const router = express.Router()
+
+router.get('articles/article/:id', (req, res) => {
+    const db = req.app.get('db')
+    db.query('SELECT * FROM article WHERE id = ' + req.params.id, (error, results) => {
```

Comment:
This is actually cool. Concatenating the string like this allows people to run malicious SQL code on our DB. Check [SQL Injection](https://www.w3schools.com/sql/sql_injection.asp) if you're interested.

If I call: `/articles/article/150 OR 1=1`, I will get all the articles. Of course, I can run any SQL statement. Something like:
`/articles/article/150; DROP TABLE articles` will delete the whole table.

The [documentation](https://github.com/mysqljs/mysql#preparing-queries) shows you how to add variables safely.
```suggestion
    const sql = mysql.format('SELECT * FROM article WHERE id = ??', [req.params.id])
    db.query(sql, (error, results) => {
```

Ask if its not clear.